lambdas/es/indexer/index.py

```diff
@@ -424,11 +424,12 @@ def handler(event, context):
                     size=size
                 )
                 # decode Quilt-specific metadata
-                try:
-                    if "helium" in meta:
-                        meta["helium"] = json.loads(meta["helium"])
-                except (KeyError, json.JSONDecodeError):
-                    print("Unable to parse Quilt 'helium' metadata", meta)
+                if meta and "helium" in meta:
+                    try:
+                        decoded_helium = json.loads(meta["helium"])
+                        meta["helium"] = decoded_helium if decoded_helium else {}
```

Comment:
`... = decoded_helium or {}`

Reply:
Also, might be outside the scope of this PR, but would be nice to clean it up: instead of sticking decoded JSON back into `meta["helium"]`, just pass it as a separate variable. (`transform_meta` doesn't even touch any metadata besided `helium`...)